src/components/form/PrintPDF.js

```diff
@@ -103,13 +109,13 @@ export default class PrintPDF extends  React.Component
     }
 
     //recoveries
-    var recoveryDescription = doc.setFontSize(8).splitTextToSize(recovery_description, 90)
+    var recoveryDescription = createParagraph(doc, recovery_description, 90)
```

Comment:
We may need to modify the createParagraph function to pass an optional maxHeight that will cutoff the text if it exceeds a certain height. I believe if they currently enter a long recoveryDescription, the text will just continue down the document even if it exceeds the height we would like. This would be a great thing to look into!

Reply:
I made it so the descriptions can only go three lines. This method is called somewhere else, we can make adjustments  if needed.